### DISCUSSION TITLE
如何对双输入pt模型进行Translator的构造

### DISCUSSION THREAD

**Discussion by KingWaq** [deepjavalibrary/djl#2612] [Q&A]
在使用pt模型时，模型的输入为model(float[][][] input1,int[][][] input2)，可是在进行Translator构造时，只能有一种输入（Translator<I, O>）无法满足模型双输入，请问这种情况该如何处理，任何帮助都不胜感激

**Comment by frankfliu** [comment]
You create a simple java Object as Input:

```
class Data {
    float[][][] input1,
    int[][][] input2
}
```

Or use built-in class: `ai.djl.util.Pair<float[][][], int[][][]>`
